### DISCUSSION TITLE
swipe to refresh

### DISCUSSION THREAD

**Discussion by commovere** [stevengharris/SplitView#28] [Q&A]
I have a screen with vsplit 
```swift
struct HomeScreen: View {

    var body: some View {
        VSplit(
            top: { HomeTopView() },
            bottom: { HomeBottomView() }
        )
        .fraction(0.3)
        .constraints(minPFraction: 0.1, minSFraction: 0.7)
        .splitter{Splitter.invisible()}
    }
}
```

I want to drag down and at bottom point (`minSFraction: 0.7`) it bounce calling some action. It should look like table swipe to refresh functionality. Is it possible now?

**Comment by stevengharris** [comment]
Currently you would need to do this via `onDrag` and then test on the fraction it passes back. See https://github.com/stevengharris/SplitView#monitoring-and-responding-to-splitter-movement
